ui/config/index.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
   },
   dev: {
     env: require('./dev.env'),
-    port: 8080,
+    port: 8090,
```

Comment:
What is this port change about? Just curious

Reply:
Ah.. it's not directly related to this PR, but  I had to update it so I can test the code. This port is used to run the local dev instance (by running.. npm run dev) and 8080 is the default port that many other localhost web servers use. I had another UI running on my machine using 8080, so I had to change the port for this app before I can test this PR. 

Sorry for another sloppy git commit..!!

Reply:
Thank you for the explanation!
We are all yet to raise up to @kyleam 's standard in commits and PR composition ;-)